src/components/FullWebsite.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import Home from './Home';
+import QuoteApi from './NinjaAPI';
+import Contact from './Contact';
+import Calculator from './Calculator';
+
+import '../Styles/Css/App.css';
+
+function FullWebsite() {
+  const [resourceType, setResourceType] = useState('Home');
+
+  const renderComponent = () => {
+    if (resourceType === 'QuoteApi') {
+      return <QuoteApi />;
+    }
+    if (resourceType === 'Contact') {
+      return <Contact />;
+    }
+    if (resourceType === 'Home') {
+      return <Home />;
+    }
+    if (resourceType === 'Calculator') {
+      return <Calculator />;
+    }
+
+    return (
+      <div className="mainSect">
+        {' '}
+        <Home />
+      </div>
+    );
+  };
+
+  return (
+    <>
+      <header>
+        <nav>
+          <div className="navBar">
+            <h2>Geeky Calculator</h2>
+            <div>
+              <button type="button" className="link" id="addNewBook" onClick={() => setResourceType('Home')}>
+                Home
+              </button>
+              <span>|</span>
+              <button type="button" className="link" id="addNewBook" onClick={() => setResourceType('Calculator')}>
+                Calculator
+              </button>
+              <span>|</span>
+              <button type="button" className="link" id="listBtn" onClick={() => setResourceType('QuoteApi')}>
+                Quotes
+              </button>
+              <span>|</span>
+              <button type="button" className="link" id="addNewBook" onClick={() => setResourceType('Contact')}>
+                Contact
+              </button>
+            </div>
+          </div>
+        </nav>
+      </header>
+      <div className="mainSect">
+        {/* Render the selected component */}
+        {renderComponent()}
+      </div>
+    </>
```

Comment:
- [ ] According to the project requirement for every page, there should be a dedicated route here is the link by [React Router V6](https://reactrouter.com/). It is most likely the implementation gets done in the `App.js` component. Here is the example how you can implement it:
``````jsx
function App() {
  return (
    <div className="app">
      <BrowserRouter>
        <Routes>
          <Route path="/" element={<Layout />}>
            <Route index element={<Home />} />
            <Route path="/calculator" element={<Calculator />} />
            <Route path="/quote" element={<Quotes />} />
          </Route>
        </Routes>
      </BrowserRouter>
    </div>
  );
}
```